_sources/Sequences/StringsandLists.rst

```diff
@@ -65,6 +62,35 @@ another list.
     should generally not mix types together. A list of just strings or just integers or just floats is generally 
     easier to deal with.
 
+Tuples
+------
+
+A **tuple**, like a list, is a sequence of items of any type. The printed representation of a tuple is a comma-separated 
+sequence of values, enclosed in parentheses. In other words, the representation is just like lists, except with 
+parentheses () instead of square brackets [].
+
+One way to create a tuple is to write an expression, enclosed in parentheses,
+that consists of multiple other expressions, separated by commas.
+
+.. sourcecode:: python
+
+    julia = ("Julia", "Roberts", 1967, "Duplicity", 2009, "Actress", "Atlanta, Georgia")
+
+Tuples are useful for representing what other languages often call *records* --- some related information that belongs 
```

Comment:
Please omit this paragraph. Nothing special about tuples for storing records; could be lists just as well.

Please add:
The key difference between lists and tuples is that a tuple is *immutable*, meaning that its contents can't be changed after the tuple is created. We will examine the mutability of lists in detail in the chapter on `Mutability`_.